invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -66,13 +70,13 @@
   </div>
 </div>
 
-<section class="ui sixteen wide column content" id="{{ group }}-affiliations" aria-label="{{ _('Affiliations for') }} {{ group }}">
+<section class="ui sixteen wide column content affiliations-list" id="{{ group }}-affiliations" aria-label="{{ _('Affiliations for') }} {{ group }}">
   <dl>
   {% for affiliation in affiliations %}
-      <dd class="text-muted">
-          {{ affiliation[0] }}.
+      <dd>
+          {{- affiliation[0] }}.
 
-          {%- if affiliation[2] %}
+          {% if affiliation[2] %}
```

Comment:
minor: would flip these `-` if possible as per above (won't repeat comment for other instances).